coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentControllerBase.java

```diff
@@ -354,7 +355,7 @@ protected ModelAndView navigate(ProposalDevelopmentDocumentForm form, BindingRes
          boolean canEdit = form.isCanEditView();
          if (isNavigateAwayFromAttachment(navigateToPageId, form.getPageId())) {
              prepareLocks(form);
-             return narrativePageSave(form, canEdit);
+             return narrativePageSaveOnNavigate(form, canEdit);
```

Comment:
I don't think these 2 methods do anything significantly different. The jira specifically says that the problem occurs when you click save on the attachments page before navigating away. Are you sure the issue isn't because we only handle the different locking sections of the proposal on navigate and ignore it when we save?